### PR TITLE
Cosmetic adjustment + code cleanup for article/mod page

### PR DESCRIPTION
**Pull request by benhalpern**
## What type of PR is this? (check all applicable)

- [x] Refactor
- [x] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description
Follow up on #5017 to get the styling right.

Also removed some hacks to make this area of the code _a bit cleaner_.
Screen Shot 2019-12-13 at 1 44 09 PM

### CODE REVIEW

File: app/views/moderations/mod.html.erb [hunk 1]
```diff
@@ -245,15 +104,21 @@
...
       <% end %>
-      <p style="width: 90%;margin:50px auto;text-align:left">
-        <b style="font-size:1.1em">All rating votes are private.</b>
-        <br><br>Provide a rating between 1 and 10 based on which audience would find this post most valuable.
-        <br><br>1: for brand new developers, 3: for junior developers, 10: for very advanced developers, etc.
-        <br><br>This will provide
-        <em>approximate and gentle</em> algorithmic adjustments to help deliver relevant content.
-        <br><br>It will be used as one of many indicators. No one rating will have an overly dramatic affect.
+      <p>
+        <br><br>Who will find this post most valuable, based on experience level?
```
Comment (jacobherrington) [inline app/views/moderations/mod.html.erb]: The line break tag `<br>` is [considered deprecated and a potential frustration for screen readers[(https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br). Do you think we could get rid of it and use multple paragraph tags instead?

File: app/views/moderations/mod.html.erb [hunk 2]
```diff
@@ -1,127 +1,10 @@
...
-    display: block;
-    font-size: 20px;
-    padding: 12px;
-    font-weight: bold;
-    margin: auto;
-    border: solid 2px;
-    margin-top: 5px;
-  }
-</style>
+<% cache "mod-styling-#{ApplicationConfig['HEROKU_SLUG_COMMIT']}" do %>
```
Comment (jacobherrington) [inline app/views/moderations/mod.html.erb]: Are there more places we can do this? If so, we could drop this into a partial, probably.

File: app/views/moderations/mod.html.erb (hunk 1 above)
Comment (benhalpern) [reply app/views/moderations/mod.html.erb]: Great call. I typically think of BR tags as a total hack to accomplish things, but this is further reason to get them out.

Comment (jacobherrington) [reply app/views/moderations/mod.html.erb]: I think for short term stuff they can be fine, especially in places like internal, but they should only be used for prototyping imo. Even then I try not to use them.

**Pull request merged**